pkg/kor/kor_test.go

```diff
@@ -60,3 +61,62 @@ func TestCalculateResourceDifference(t *testing.T) {
 		}
 	}
 }
+
+func getFakeConfigContent() string {
+	fakeContent := `
+apiVersion: v1
+clusters:
+- cluster:
+    server: https://localhost:8080
+    extensions:
+    - name: client.authentication.k8s.io/exec
+      extension:
+        audience: foo
+        other: bar
+  name: foo-cluster
+contexts:
+- context:
+    cluster: foo-cluster
+    namespace: bar
+  name: foo-context
+current-context: foo-context
+kind: Config
+`
+	return fakeContent
+}
+
+func TestGetKubeClient_fromEnvVar(t *testing.T) {
+	configFile, err := os.CreateTemp("", "kubeconfig-")
+	if err != nil {
+		t.Error(err)
+	}
+	defer os.Remove(configFile.Name())
+	if err := os.WriteFile(configFile.Name(), []byte(getFakeConfigContent()), 0666); err != nil {
+		t.Error(err)
+	}
+
+	originalKCEnv := os.Getenv("KUBECONFIG")
+	defer os.Setenv("KUBECONFIG", originalKCEnv)
+	os.Setenv("KUBECONFIG", configFile.Name())
+
+	kcs := GetKubeClient("")
+	if kcs == nil {
+		t.Errorf("Expected valid clientSet")
+	}
+}
+
+func TestGetKubeClient_fromInput(t *testing.T) {
```

Comment:
Please rename the function to use camelCase

Reply:
done